ImplementationOfFinishedProducts/ImplementationOfFinishedProductsUnitTest/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using ImplementationOfFinishedProducts;
+using NUnit.Framework;
+
+namespace ImplementationOfFinisedProductsUnitTest
+{
+    class Program
```

Comment:
Класс, содержащий тесты должен быть отрытым и маркироваться соответствующим атрибутом. И лучше его переименовать, чтобы было понятно, тесты методов какого класса он содержит.
```suggestion
    [TestFixture]
    public class Commodity.UnitTests
```